config/rbac/role.yaml

```diff
@@ -3,6 +3,18 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: manager-role
+aggregationRule:
```

Comment:
I was actually also planning to make this RBAC change soon. Depending on how long you think the rest of this PR might take, a separate PR with this particular change would be welcome!

Reply:
Hey @nojnhuh, thanks, I'm also thinking about opening aggregated role PR separately, as the current state passes CI. Do you know how the codecov job can be retriggered? The failure there seems like a flake.